frame/staking/src/tests.rs

```diff
@@ -3213,18 +3219,11 @@ fn slash_kicks_validators_not_nominators_and_disables_nominator_for_kicked_valid
 		assert_eq!(Balances::free_balance(11), 900);
 		assert_eq!(Balances::free_balance(101), 2000 - nominator_slash_amount_11);
 
-		// This is the best way to check that the validator was chilled; `get` will
-		// return default value.
-		for (stash, _) in <Staking as Store>::Validators::iter() {
-			assert!(stash != 11);
-		}
+		// check that validator was chilled.
+		assert!(<Staking as Store>::Validators::iter().all(|(stash, _)| stash != 11));
 
-		let nominations = <Staking as Store>::Nominators::get(&101).unwrap();
-
-		// and make sure that the vote will be ignored even if the validator
-		// re-registers.
-		let last_slash = <Staking as Store>::SlashingSpans::get(&11).unwrap().last_nonzero_slash();
-		assert!(nominations.submitted_in < last_slash);
+		// and make sure the nominator's approval vote is dropped.
```

Comment:
What is an approval vote here?

Reply:
An "approval" is one item in the list of nominations that a nominator puts forth.